truenas/webui
# NAS-132458 / 25.04 / Expose dedup_table* fields in UI

**Changes:**

Adds new line in ZFS Health card.
<img width="461" alt="Снимок экрана 2024-12-15 в 17 10 22" src="https://github.com/user-attachments/assets/72f411a9-7f31-4b8e-997d-5ce99cf4eaca" />


**Testing:**

1. Create a dataset with deduplication On.
2. Go to shell to this dataset in `/mnt/pool/dataset`.
3. Create a file, for example by typing in `nano`.
4. Copy file multiple times.
5. The line will appear on ZFS Health card.
